content.js

```diff
@@ -1,161 +1,193 @@
-(function() { 
+/* global chrome, PHRASES_TO_HIDE */
 
-  let isActive = true
-  let numEventsHidden = 0
-  let timeline
-  let pageUrl
-  let monitorInterval
-  const observer = new MutationObserver(checkForTimelineChanges)
+(function () {
+  const itemClass = 'TimelineItem';
+  const itemHiddenClass = 'TimelineItem-Hidden';
 
-  function checkForTimelineChanges(mutations) {
-    // when the dom has changed - check if the timeline has been added to
-    const timelineChanges = mutations.find((mutation) => {
-      return [...mutation.addedNodes].map(m => m.className && m.className.includes('js-timeline-item')).includes(true)
-    })
-    if (timelineChanges) processTimeline()
-  }
+  let isActive = true;
```

Comment:
What does isActive mean? Does it mean that the button has been clicked and stuff is happening? Or does it mean that loading has finished?

I wonder if it would be better to have a variable that describes a state like

state = 'idle' | 'loading' | 'complete'

Or is it totally separate?